client/users/user-entries.ts

```diff
@@ -10,6 +10,10 @@ export type UserEntry = [userId: SbUserId, username: string | undefined]
  * Returns a function for use with `useAppSelector` that maps the given `userIds` to their
  * associated names. Should be used with `areUserEntriesEqual` when passed to `useAppSelector`.
  *
+ * The user entries are sorted because the `areUserEntriesEqual` equality function this selector is
```

Comment:
Just say that this function returns the user entries sorted alphabetically, and that the sort is stable.